package.json

```diff
@@ -9,9 +9,19 @@
     "react-router-dom": "^4.2.2"
   },
   "devDependencies": {
+    "eslint": "^4.9.0",
+    "eslint-config-airbnb": "^16.1.0",
+    "eslint-config-prettier": "^2.9.0",
+    "eslint-plugin-import": "^2.7.0",
+    "eslint-plugin-jsx-a11y": "^6.0.2",
+    "eslint-plugin-prettier": "^2.6.0",
+    "eslint-plugin-react": "^7.4.0",
+    "prettier": "^1.11.1",
     "react-scripts": "1.1.1"
   },
   "scripts": {
+    "prettier": "./node_modules/.bin/prettier --write 'src/**/*.jsx' 'src/**/*.js'",
```

Comment:
Can't you just use "prettier"? It'll pick up things in node_modules/.bin/prettier automatically

Reply:
Yeah, you can. I just always try and be specific.